DashSync/shared/Models/Transactions/Base/DSTransaction.h

```diff
@@ -138,6 +131,7 @@ typedef union _UInt160 UInt160;
 - (BOOL)signWithPrivateKeys:(NSArray *)keys;
 - (BOOL)signWithPreorderedPrivateKeys:(NSArray *)keys;
 
+- (NSUInteger)masterNodeOutputIndex;
```

Comment:
This should not be on DSTransaction, as only provider registration transactions can have it. (also please call it masternodeOutputIndex and not masterNodeOutputIndex)